types/Accounts.ts

```diff
@@ -148,32 +82,6 @@ export type EnabledFeaturesResponse = {
   enabledFeatures: { [key: string]: boolean };
 };
 
-export type UpdateAccountConfigOptions =
-  Partial<FlatAccountFields_DEPRECATED> & {
-    environment?: Environment;
-  };
-
-export type PersonalAccessKeyOptions = {
-  accountId: number;
-  personalAccessKey: string;
-  env: Environment;
-};
-
-export type OAuthOptions = {
-  accountId: number;
-  clientId: string;
-  clientSecret: string;
-  refreshToken: string;
-  scopes: Array<string>;
-  env: Environment;
-};
-
-export type APIKeyOptions = {
-  accountId: number;
-  apiKey: string;
-  env: Environment;
-};
-
```

Comment:
Anywhere that previously used these types now just use the accounts themselves to keep things simple